### ISSUE TITLE
omnibox navigation

### ISSUE THREAD

**Issue by schmeic** [pragmader/navimacs#10]
The one thing from firemacs that I use more than anything else is navigating next and previous suggestions from the omnibox using C-n and C-p. I've been looking at the WebExtensions API, and I'm not even sure if this is possible. It seems like using C-s to open the Find box may also not be possible, or am I missing something?

In both cases, I guess the extension could open a popup and duplicate existing browser functionality, but this seems like a lot of work just to add new keybindings. I'm willing to help out with this is you have any ideas. Thanks!

**Comment by pragmader** [comment]
Yeah, I was thinking about C-n/C-p navigation in the omnibox myself. And I will definitely do that once I have time. But not sure about the rest of the stuff sounds like too much effort for this I'd expect some help from external contributors as well.

**Comment by schmeic** [comment]
I am almost positive now (after more research) that it is not possible to use WebExtensions to control the omnibox navigation.
